content/v2.0/reference/api/client-libraries/python.md

```diff
@@ -103,3 +99,70 @@ write_api = client.write_api(write_options=SYNCHRONOUS)
 p = influxdb_client.Point("my_measurement").tag("location", "Prague").field("temperature", 25.3)
 write_api.write(bucket=bucket, org=org, record=p)
 ```
+## Query data from InfluxDB with Python
+
+In order to query data, we need to instantiate the query client. 
+
+```python
+query_api = client.query_api()
+```
+
+Next, we create a flux query. 
+
+```python
+query = ‘ from(bucket:"my-bucket")\
+|> range(start: -10m)\
+|> filter(fn:(r) => r._measurement == “my_measurement”)\
+|> filter(fn: (r) => r.location == Prague")\
+|> filter(fn:(r) => r._field == "temperature" )‘
+```
+
+We query the InfluxDB server with our flux query. The query client returns the results as a Flux Object with a table structure. The query() method takes two parameters: `org` and `query`.  
```

Comment:
The **query client** sneds the Flux query to InfluxDB and returns a Flux Object with a table structure. The `query()` method...